website/snippets/_sl-partner-links.md

```diff
@@ -32,11 +32,11 @@ The following tools integrate with the dbt Semantic Layer:
 
 <div className="card-container">
   <Card
-    title="KlipFolio Power Metrics"
-    body="Learn how to connect, and deliver a streamlined metrics catalog to your business users."
-    link="https://support.klipfolio.com/hc/en-us/articles/18164546900759-PowerMetrics-Adding-dbt-Semantic-Layer-metrics"
-    icon="klipfolio"/>
-    <a href="https://support.klipfolio.com/hc/en-us/articles/18164546900759-PowerMetrics-Adding-dbt-Semantic-Layer-metrics"
+    title="Klipfolio PowerMetrics"
+    body="Learn how to connect to a streamlined metrics catalog and deliver metric-centric analytics to business users."
+    icon="klipfolio"
+    link="https://www.klipfolio.com/dbt"/>
```

Comment:
This link takes the user to a sales/marketing landing page rather than an instructional page (the old link leads to instructions). The other cards on this page lead to instructional material. Is this a change we want to make in our docs?